sup3r/models/surface.py

```diff
@@ -567,8 +567,9 @@ def meta(self):
                 'weight_for_delta_topo': self._w_delta_topo,
                 'pressure_divisor': self._pres_div,
                 'pressure_exponent': self._pres_exp,
-                'training_features': self.training_features,
-                'output_features': self.output_features,
+                'features': self._features,
```

Comment:
I'm open to discussion on this. For the linear and surface models i left "features" as the default input because these models are simple parameterizations and should(?) always have the same input vs output features so i thought "features" as the input arg/meta param made sense. But like i said... open to discussion.

Reply:
Yeah maybe it makes the most sense to have `lr_features` as input and keep `hr_out_features` as a property that is basically an alias and remove the generic `features`? This is the debate i had with myself yesterday and gave up haha.

Reply:
Yeah thats what I was thinking as a solution also.

Reply:
okay cool will do!